packages/openneuro-components/src/dataset/ModalitiesMetaDataBlock.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Markdown from 'markdown-to-jsx'
+
+export interface ModalitiesMetaDataBlockProps {
+  heading: string
+  isMarkdown?: boolean
+  item: React.ReactNode | string[] | number
+  className?: string
+}
+
+export const ModalitiesMetaDataBlock = ({
+  heading,
+  isMarkdown,
+  item,
+  className,
+}: ModalitiesMetaDataBlockProps) => {
+  const arrayToMarkdown = arr => {
+    return arr
+      ? arr
+          .map(
+            element =>
+              ` * <a href="/search/${element.toLowerCase()}">${element}</a>\n`,
+          )
+          .join('')
+      : ''
+  }
+  return (
+    <div className={'dataset-meta-block ' + className}>
+      <h2 className="dmb-heading">{heading}</h2>
+      {isMarkdown ? (
+        <Markdown>{arrayToMarkdown(item) || 'N/A'}</Markdown>
+      ) : (
+        item || 'N/A'
+      )}
```

Comment:
Instead of going through markdown-to-jsx, we could just use jsx. This would also let us use <Link/> from react-router-dom instead of anchor tags.